Modularity: Update structure (v25.1) (#8434)
DevExpress/devextreme-documentation: concepts/Common/Modularity/02 DevExtreme Modules Structure/file_management/upload_info.md

> Exposes an object that offers information about the file upload session.

Reviewer: ```suggestion
Exposes an object with information about the file upload session.
```
